### PR TITLE
Added get_discovery_image post upgrade task

### PR DESCRIPTION
**Pull request by ntkathole**

### CODE REVIEW

**Comment by san7ket** [comment]
@ntkathole so this is not a job now , and the task will run for every pipeline and rhel distro, @sghai will the commands in the task work for every pipeline, 6.1 6.2 and 6.3.

**Comment by ntkathole** [comment]
@san7ket yes the task should work for every pipeline and distro.

File: upgrade/helpers/tasks.py [hunk 1]
```diff
@@ -345,6 +346,12 @@ def post_upgrade_test_tasks(sat_host, cap_host=None):
         sat_version=sat_version,
         host=sat_host
     )
+    # Execute task for creating latest discovery iso required for unattended
+    #  test
+    execute(
+        get_discovery_image,
```
Comment (san7ket) [inline upgrade/helpers/tasks.py]: We will also need BeautifulSoup as requirement for this ryt?

**Comment by san7ket** [comment]
Also , https://github.com/SatelliteQE/automation-tools/blob/master/automation_tools/utils.py#L106 , where do we get the IPADDR variable ?

**Comment by san7ket** [comment]
Apart for the pending questions, ACK

File: upgrade/helpers/tasks.py (hunk 1 above)
Comment (ntkathole) [reply upgrade/helpers/tasks.py]: @san7ket We will get that from https://github.com/SatelliteQE/automation-tools/blob/master/setup.py#L24

**Comment by ntkathole** [comment]
@san7ket We get IPADDR and other environment variables from source `provisioning_environment.conf`  in case of regular automation, it was missing for upgrade(source `sat6_upgrade.conf ` ) Raised gitlab PR for that. Thanks.

**Pull request merged**